multiaccounts/accounts/database.go

```diff
@@ -52,22 +52,24 @@ type Keypair struct {
 }
 
 type Account struct {
-	Address   types.Address             `json:"address"`
-	KeyUID    string                    `json:"key-uid"`
-	Wallet    bool                      `json:"wallet"`
-	Chat      bool                      `json:"chat"`
-	Type      AccountType               `json:"type,omitempty"`
-	Path      string                    `json:"path,omitempty"`
-	PublicKey types.HexBytes            `json:"public-key,omitempty"`
-	Name      string                    `json:"name"`
-	Emoji     string                    `json:"emoji"`
-	ColorID   common.CustomizationColor `json:"colorId,omitempty"`
-	Hidden    bool                      `json:"hidden"`
-	Clock     uint64                    `json:"clock,omitempty"`
-	Removed   bool                      `json:"removed,omitempty"`
-	Operable  AccountOperable           `json:"operable"` // describes an account's operability (read an explanation at the top of this file)
-	CreatedAt int64                     `json:"createdAt"`
-	Position  int64                     `json:"position"`
+	Address               types.Address             `json:"address"`
+	KeyUID                string                    `json:"key-uid"`
+	Wallet                bool                      `json:"wallet"`
+	Chat                  bool                      `json:"chat"`
+	Type                  AccountType               `json:"type,omitempty"`
+	Path                  string                    `json:"path,omitempty"`
+	PublicKey             types.HexBytes            `json:"public-key,omitempty"`
+	Name                  string                    `json:"name"`
+	Emoji                 string                    `json:"emoji"`
+	ColorID               common.CustomizationColor `json:"colorId,omitempty"`
+	Hidden                bool                      `json:"hidden"`
+	Clock                 uint64                    `json:"clock,omitempty"`
+	Removed               bool                      `json:"removed,omitempty"`
+	Operable              AccountOperable           `json:"operable"` // describes an account's operability (read an explanation at the top of this file)
+	CreatedAt             int64                     `json:"createdAt"`
+	Position              int64                     `json:"position"`
+	ProdPreferredChainIDs string                    `json:"prodPreferredChainIds"`
```

Comment:
I'd use a slice of uint64s everywhere other than the DB instead of moving around the encoded string